src/llmcompressor/entrypoints/model_free/process.py

```
@@ -253,3 +256,63 @@ def _load_tensors_from_inverse_weights_map(
+                raise ValueError(
+                    f"{unsplit_name} expects a second dimension divisible by "
+                    f"{len(split_names)} but shape: {tensor.shape}"
+                )
+
+            # Split into experts
+            intermediate_size = tensor.shape[1] // len(split_names)
+            for expert_idx in range(num_experts):
+                expert_tensor = tensor[expert_idx]
+                # Split into layers
+                split_layers = expert_tensor.split(intermediate_size, dim=0)
+                for split_name, split_layer in zip(split_names, split_layers):
```

Ruff (0.15.9)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)